### PR TITLE
fix(sdk): include ontology_type_schemas/*.yaml in setuptools

### PR DESCRIPTION
**Pull request by chensun**
**Description of your changes:**
Follow up on #5197 to include the schema yaml files into setuptools
Fixes https://github.com/kubeflow/pipelines/issues/5240

**Checklist:**
- [x] The title for your pull request (PR) should follow our title convention. [Learn more about the pull request title convention used in this repository](https://github.com/kubeflow/pipelines/blob/master/CONTRIBUTING.md#pull-request-title-convention). 

- [ ] Do you want this pull request (PR) cherry-picked into the current release branch?

    [Learn more about cherry-picking updates into the release branch](https://github.com/kubeflow/pipelines/blob/master/RELEASE.md#cherry-picking-pull-requests-to-release-branch).

### CODE REVIEW

File: sdk/python/MANIFEST.in [hunk 1]
```diff
@@ -0,0 +1 @@
+include kfp/dsl/ontology_type_schemas/*.yaml
```
Comment (neuromage) [inline sdk/python/MANIFEST.in:1]: can this directory be renamed to just type_schemas (drop the redundant `ontology`)?

/cc @chensun 
/cc @dushyanthsc

Comment (chensun) [reply sdk/python/MANIFEST.in:1]: Chatted with @dushyanthsc, he'll make the change.

Comment (neuromage) [reply sdk/python/MANIFEST.in:1]: I noticed it was called `schema` in a current PR, that LGTM as well. Thanks!

**Pull request merged**